Makefile

```diff
@@ -131,11 +131,13 @@ deploy: manifests ## Deploy controller in the configured Kubernetes cluster in ~
 .PHONY: manifests
 manifests: ## Generate manifests e.g. CRD, RBAC etc.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
AFAIK everything we need to generate is always in the apis directory, isn't it? I think we could remove this line (and the equivalent below).
That would at least solve the problem of trying to keep the controller-gen version the same in two different go.mod files.

Reply:
makes sense. @s3rj1k WDYT?

Reply:
https://github.com/metal3-io/baremetal-operator/pull/888/files